### PR TITLE
docs: Add doc for scanners that report individual layers

### LINKED ISSUE #764: [DOC] Explain nuance when scanner scans all the layers that would include vulns that have been patched by the top layer
### What kind of documentation improvement is needed?

Missing information or guidance

### What is the change that is needed?

_No response_

### Are you willing to submit PRs to contribute to documentation?

- [ ] Yes, I am willing to implement it.

### PR DESCRIPTION
**Pull request by ritazh**
Describe the changes in this pull request using active verbs such as _Add_, _Remove_, _Replace_ ...

Closes #764

### CODE REVIEW

**Comment by ritazh** [comment]
will update version docs after the change is reviewed

File: website/docs/faq.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ RPM: All RPM-based images will use `mcr.microsoft.com/cbl-mariner/base/core:2.0`
 
 APK: APK-based images never use a tooling image, as Copa does not patch distroless alpine images.
 
+## After Copa patched the image, why does the scanner still show patched vulnerabilities
```
Comment (sozercan) [inline website/docs/faq.md:34]:
```suggestion
## After Copa patched the image, why does the scanner still show patched OS packages vulnerabilities?
```

**Review by sozercan — APPROVED** [review]
lgtm

**Review by ashnamehrotra — APPROVED** [review]
LGTM

**Pull request merged**